### ISSUE TITLE
Invalid mcMMO skill ()

### ISSUE THREAD

**Issue by myfbone** [Rocologo/MobHunting#550]
```
[18:51:14 WARN]: [mcMMO] Invalid mcMMO skill ()
[18:51:14 ERROR]: Could not pass event EntityDeathEvent to MobHunting v6.8.3
com.gmail.nossr50.api.exceptions.InvalidSkillException: That is not a valid skill.
        at com.gmail.nossr50.api.ExperienceAPI.getSkillType(ExperienceAPI.java:1142) ~[?:?]
        at com.gmail.nossr50.api.ExperienceAPI.addLevel(ExperienceAPI.java:576) ~[?:?]
        at one.lindegaard.MobHunting.compatibility.McMMOCompat.addLevel(McMMOCompat.java:108) ~[?:?]
        at one.lindegaard.MobHunting.compatibility.McMMOCompatHelper.addLevel(McMMOCompatHelper.java:30) ~[?:?]
        at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1793) ~[?:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.14.4.jar:git-Paper-162]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.4.jar:git-Paper-162]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-162]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-162]
        at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:741) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityLiving.d(EntityLiving.java:1419) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityLiving.die(EntityLiving.java:1354) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityLiving.damageEntity(EntityLiving.java:1198) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityMonster.damageEntity(EntityMonster.java:56) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityZombie.damageEntity(EntityZombie.java:284) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:376) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:288) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityArrow.tick(EntityArrow.java:199) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityTippedArrow.tick(EntityTippedArrow.java:87) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.WorldServer.entityJoinedWorld(WorldServer.java:584) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.World.a(World.java:936) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.WorldServer.doTick(WorldServer.java:356) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1207) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-162]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by Rocologo** [comment]
please give me a few more lines... before the error and after th eerror (including caused by)

I so tired of the mess with McMMO and McMMO Classic :-(

**Comment by Rocologo** [comment]
I wunder why itr says, before the error???

[mcMMO] Invalid mcMMO skill ()

**Comment by myfbone** [comment]
```
[18:24:32] [Server thread/INFO]: CONSOLE issued server command: /give Terrebat gold_ingot 1
[18:24:32] [Server thread/INFO]: Выдано 1 штук gold ingot игроку Terrebat.
[18:24:43] [Server thread/INFO]: CONSOLE issued server command: /give KVADROGOLOVIK string 4
[18:24:43] [Server thread/INFO]: Выдано 4 штук string игроку KVADROGOLOVIK.
[18:24:43] [Server thread/INFO]: CONSOLE issued server command: /give KVADROGOLOVIK spider_eye 10
[18:24:43] [Server thread/INFO]: Выдано 10 штук spider eye игроку KVADROGOLOVIK.
[18:24:46] [Server thread/WARN]: [mcMMO] Invalid mcMMO skill ()
[18:24:46] [Server thread/ERROR]: Could not pass event EntityDeathEvent to MobHunting v6.8.3
com.gmail.nossr50.api.exceptions.InvalidSkillException: That is not a valid skill.
	at com.gmail.nossr50.api.ExperienceAPI.getSkillType(ExperienceAPI.java:1142) ~[?:?]
	at com.gmail.nossr50.api.ExperienceAPI.addLevel(ExperienceAPI.java:576) ~[?:?]
	at one.lindegaard.MobHunting.compatibility.McMMOCompat.addLevel(McMMOCompat.java:108) ~[?:?]
	at one.lindegaard.MobHunting.compatibility.McMMOCompatHelper.addLevel(McMMOCompatHelper.java:30) ~[?:?]
	at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1793) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.14.4.jar:git-Paper-162]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:741) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.d(EntityLiving.java:1419) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.die(EntityLiving.java:1354) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.damageEntity(EntityLiving.java:1198) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityMonster.damageEntity(EntityMonster.java:56) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:376) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:288) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.tick(EntityArrow.java:199) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityTippedArrow.tick(EntityTippedArrow.java:87) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.WorldServer.entityJoinedWorld(WorldServer.java:584) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.World.a(World.java:936) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.WorldServer.doTick(WorldServer.java:356) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1207) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-162]
	at java.lang.Thread.run(Thread.java:834) [?:?]
[18:25:11] [Server thread/WARN]: [mcMMO] Invalid mcMMO skill ()
[18:25:11] [Server thread/ERROR]: Could not pass event EntityDeathEvent to MobHunting v6.8.3
com.gmail.nossr50.api.exceptions.InvalidSkillException: That is not a valid skill.
	at com.gmail.nossr50.api.ExperienceAPI.getSkillType(ExperienceAPI.java:1142) ~[?:?]
	at com.gmail.nossr50.api.ExperienceAPI.addLevel(ExperienceAPI.java:576) ~[?:?]
	at one.lindegaard.MobHunting.compatibility.McMMOCompat.addLevel(McMMOCompat.java:108) ~[?:?]
	at one.lindegaard.MobHunting.compatibility.McMMOCompatHelper.addLevel(McMMOCompatHelper.java:30) ~[?:?]
	at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1793) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.14.4.jar:git-Paper-162]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:741) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.d(EntityLiving.java:1419) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.die(EntityLiving.java:1354) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.damageEntity(EntityLiving.java:1198) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityMonster.damageEntity(EntityMonster.java:56) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityZombie.damageEntity(EntityZombie.java:284) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:376) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:288) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.tick(EntityArrow.java:199) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityTippedArrow.tick(EntityTippedArrow.java:87) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.WorldServer.entityJoinedWorld(WorldServer.java:584) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.World.a(World.java:936) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.WorldServer.doTick(WorldServer.java:356) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1207) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-162]
	at java.lang.Thread.run(Thread.java:834) [?:?]
[18:25:11] [Server thread/WARN]: [mcMMO] Invalid mcMMO skill ()
[18:25:11] [Server thread/ERROR]: Could not pass event EntityDeathEvent to MobHunting v6.8.3
com.gmail.nossr50.api.exceptions.InvalidSkillException: That is not a valid skill.
	at com.gmail.nossr50.api.ExperienceAPI.getSkillType(ExperienceAPI.java:1142) ~[?:?]
	at com.gmail.nossr50.api.ExperienceAPI.addLevel(ExperienceAPI.java:576) ~[?:?]
	at one.lindegaard.MobHunting.compatibility.McMMOCompat.addLevel(McMMOCompat.java:108) ~[?:?]
	at one.lindegaard.MobHunting.compatibility.McMMOCompatHelper.addLevel(McMMOCompatHelper.java:30) ~[?:?]
	at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1793) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.14.4.jar:git-Paper-162]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-162]
	at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:741) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.d(EntityLiving.java:1419) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.die(EntityLiving.java:1354) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityLiving.damageEntity(EntityLiving.java:1198) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityMonster.damageEntity(EntityMonster.java:56) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityZombie.damageEntity(EntityZombie.java:284) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:376) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.a(EntityArrow.java:288) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityArrow.tick(EntityArrow.java:199) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.EntityTippedArrow.tick(EntityTippedArrow.java:87) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.WorldServer.entityJoinedWorld(WorldServer.java:584) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.World.a(World.java:936) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.WorldServer.doTick(WorldServer.java:356) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1207) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-162]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-162]
	at java.lang.Thread.run(Thread.java:834) [?:?]
[18:25:17] [Server thread/INFO]: CONSOLE issued server command: /give KVADROGOLOVIK bone 1
[18:25:17] [Server thread/INFO]: Выдано 1 штук bone игроку KVADROGOLOVIK.
```

**Comment by myfbone** [comment]
> 
> 
> I wunder why itr says, before the error???
> 
> [mcMMO] Invalid mcMMO skill ()

Guess it's because it's trying to reward player mcmmo skill points, but the skillname is incorrect?

**Comment by Rocologo** [comment]
it could be... what did you write?

**Comment by myfbone** [comment]
What do you mean what did I write?

**Comment by Rocologo** [comment]
Wel I dont understand where the Invalid Skill come come from and the Invalid skill might be the reason that MobHunting fails.

**Comment by Rocologo** [comment]
I think I know what wrong, wait...

**Comment by myfbone** [comment]
I use translation file with mcmmo and in game commands use Russian names of skills. Mb that's the problem?

**Comment by Rocologo** [comment]
Please try this snapshot

https://fractal.lindegaard.one:8181/job/MobHunting/897/

**Comment by Rocologo** [comment]
It looks to me that the error has gone, but there is still a problem. If I kill a spider with a mob, it seems to be classified as "UNARMED" and the McMMO level is given as "UNARMED", so I will investigate this again.

**Comment by Rocologo** [comment]
Please test  in v6.8.5 and let me know if it works now. I tested a lot last night and I got McMMO Level rewards for both SWORDS and AXES

**Comment by myfbone** [comment]
Think it's fixed. Do not see this messages anymore

**Comment by myfbone** [comment]
Nope. Sorry it is not
```
[19:37:17 WARN]: [mcMMO] Invalid mcMMO skill ()
[19:37:17 ERROR]: Could not pass event EntityDeathEvent to MobHunting v6.8.5
com.gmail.nossr50.api.exceptions.InvalidSkillException: That is not a valid skill.
        at com.gmail.nossr50.api.ExperienceAPI.getSkillType(ExperienceAPI.java:1142) ~[?:?]
        at com.gmail.nossr50.api.ExperienceAPI.addLevel(ExperienceAPI.java:576) ~[?:?]
        at one.lindegaard.MobHunting.compatibility.McMMOCompat.addLevel(McMMOCompat.java:117) ~[?:?]
        at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1800) ~[?:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.14.4.jar:git-Paper-162]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.4.jar:git-Paper-162]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-162]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-162]
        at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:741) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityLiving.d(EntityLiving.java:1419) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityLiving.die(EntityLiving.java:1354) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityLiving.damageEntity(EntityLiving.java:1198) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityMonster.damageEntity(EntityMonster.java:56) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityZombie.damageEntity(EntityZombie.java:284) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityHuman.attack(EntityHuman.java:1046) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.EntityPlayer.attack(EntityPlayer.java:1676) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:2031) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:51) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:6) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:1012) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:1005) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:989) ~[patched_1.14.4.jar:git-Paper-162]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:922) ~[patched_1.14.4.jar:git-Paper-162]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by Rocologo** [comment]
Do you know which weapon was used?

**Comment by myfbone** [comment]
I have no idea. But I think it is not unarmed because unarmed skill points are adding for combat

**Comment by Rocologo** [comment]
I need your help to find out which weapon cause this error.

**Comment by myfbone** [comment]
I'll try to find out tomorrow

**Comment by myfbone** [comment]
@Rocologo I think it's trident in mele attacks. BTW I also noticed that if you kill mob with trident by throwing it it will give you mcmmo unarmed level

**Comment by Rocologo** [comment]
Sorry I have not had time to code. But today I tested the Trident and you are correct. Its a problem with the Trident. Can you tell how is is classified in McMMO? I know Axe, Sword, Archery and Unarmed? Is it Archery?

**Comment by myfbone** [comment]
not sure. probably archery :relaxed:

**Comment by Rocologo** [comment]
No worries I have found a solution

**Comment by Rocologo** [comment]
Fixed in newest release 6.8.8
